charts/victoria-metrics-k8s-stack/templates/servicemonitors/nodeexporter.yaml

```diff
@@ -9,7 +9,6 @@ spec:
   selector:
     matchLabels:
       app: prometheus-node-exporter
-      app.kubernetes.io/instance: {{ default $.Release.Name $.Values.argocdReleaseOverride | quote }}
```

Comment:
i remember adding this in #197 but i never tested with this label removed. have you tested with argo @f41gh7 ?

Reply:
yep, removing it solves scraping issue.